_metadata.py

```diff
@@ -1,2 +1,2 @@
-__extension_version__ = "0.5.0rc0"
+__extension_version__ = "0.5.0"
```

Comment:
It would probably make sense to bump the version tp 0.6.0.

Reply:
updated in the @cqc-melf
[fix workflow and version in tests](https://github.com/CQCL/pytket-quantinuum/pull/3/commits/c3b9fa3e6f4c30aae59efb1492b6e80d9c5f101a)